### PR TITLE
IT-4376 | Bump FontAwesome to 6 + migrate icon names

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: lib/js/atoms/IconButton/IconButton.stories.mdx [hunk 1]
```diff
@@ -23,7 +23,7 @@ import { ICONS } from 'design-system/lib/js/atoms/Icon/Icon.consts';
            size: {control: { type: 'select', options: Object.values(ICON_BUTTON_SIZES) }, defaultValue: ICON_BUTTON_SIZES.LARGE},
            type: {control: { type: 'select', options: Object.values(ICON_BUTTON_TYPES) }, defaultValue: ICON_BUTTON_TYPES.ICON_ONLY},
            radius: {control: { type: 'select', options: Object.values(BUTTON_RADIUSES) }, defaultValue: BUTTON_RADIUSES.CAPSULE},
-           icon: {control: { type: 'select', options: Object.keys(ICONS) }, defaultValue: 'FA_TIMES_LIGHT' },
+           icon: {control: { type: 'select', options: Object.keys(ICONS) }, defaultValue: .FA_XMARK_LIGHT' },
```
Comment (damianmef) [inline lib/js/atoms/IconButton/IconButton.stories.mdx:26]:
```suggestion
           icon: {control: { type: 'select', options: Object.keys(ICONS) }, defaultValue: 'FA_XMARK_LIGHT' },
```

File: lib/js/atoms/TabItem/TabItem.stories.mdx [hunk 2]
```diff
@@ -14,7 +14,7 @@ import { ICONS } from 'design-system/lib/js/atoms/Icon/Icon.consts';
            title: 'Wpłynąłem na suchego przestówr oceanu'
 	   }}
        argTypes={{
-           icon: {control: { type: 'select', options: Object.keys(ICONS) }, defaultValue: 'FA_CALENDAR_ALT' },
+           icon: {control: { type: 'select', options: Object.keys(ICONS) }, defaultValue: .FA_CALENDAR_DAYS' },
```
Comment (damianmef) [inline lib/js/atoms/TabItem/TabItem.stories.mdx:17]:
```suggestion
           icon: {control: { type: 'select', options: Object.keys(ICONS) }, defaultValue: 'FA_CALENDAR_DAYS' },
```

Comment (kvas-damian) [reply lib/js/atoms/TabItem/TabItem.stories.mdx]: thanks

File: package.json [hunk 3]
```diff
@@ -28,6 +28,12 @@
     "synchronizeColors": "npx ts-node tools/importers/SynchronizeColorsTokens.ts"
   },
   "devDependencies": {
+    "@fortawesome/fontawesome-svg-core": "6.1.1",
```
Comment (kvas-damian) [inline package.json:31]: after moving it to `devDependencies` there is no need to install these dependencies in `wnl-platform`

**Review by pioziol123 — APPROVED** [review]

**Pull request merged**